modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-signing-certificates.adoc
+
+[id="manually-rotate-service-ca-{context}"]
+= Manually rotate the service CA certificate
```

Comment:
This really needs to be part of a separate assembly about manually rotating the certs, and I think that some of this info might need to be at the assembly level.

I think you might need a little more info about the CA certificate in general. This is the only place that it's mentioned, and I'm, again, slightly confused about its importance.

Reply:
I'd prefer to keep this in one assembly. You only need to perform this process if you utilize service certificates. These certificates are exclusively used for internal cluster communication.

The service CA certificate signs all of the service certificates, and isn't configured or touched by the customer at all. The only thing they really should know is that if they use service certificates then this **must** be rotated after a year **if and only if** they don't do an upgrade within a year. I strongly suspect less than 5% of customers will ever need to perform this, but it's something they should be aware of if they intend to use service certificates. Thus its inclusion here.